src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,27 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
```

Comment:
Your superclass is reasonable, but the subclass were defined in a incorrect way. Please approach us to clarify understanding on this topic.